msteams-platform/toolkit/build-an-AI-agent-in-Teams.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Build an AI Agent in Teams
+author: surbhigupta
+description:  In this module, learn how to build AI Agent using Teams AI library.
+ms.topic: conceptual
+ms.localizationpriority: high
+ms.author: v-ganr
+ms.date: 05/21/2024
+---
+
+# Build an AI Agent in Teams
+
+An AI agent in Microsoft Teams is a conversational chatbot that uses Large Language Models (LLMs) to interact with users. It understands user intentions and selects a sequence of actions, enabling the chatbot to complete common tasks.
```

Comment:
Should we mention this as "An AI agent bot", to make sure devs don't confuse this with OpenAI's agents, as this is a bot based solution.